README.md

```diff
@@ -70,7 +70,15 @@ yarn dev:ssl # If you are working with the login process
 > the app after login. You can either remove the trailing s in the URL after the
 > redirection, or use the `yarn dev:ssl` command to use HTTPs for the
 > development server. Also, make sure to set the `NEXTAUTH_URL` environment
-> variable to `https://localhost:3000` in your `.env.local` file.
+> variable to `https://localhost:3000` in your `.env.local` file
+
+> Authentication Requirements Checklist
+>
+> - Create .env.loval file
+> - NEXTAUTH_URL=https://localhost:3000
+> - NEXTAUTH_SECRET=Some Secret
+> - Don't forget to import all ADFS\_ from
+>   `https://vercel.com/ixt/visualization-tool/settings/environment-variables`
```

Comment:
@noahonyejese as discussed briefly in https://github.com/visualize-admin/visualization-tool/pull/1848#discussion_r1837928507, I'd try to not include IXT-specific information in README. For these variables, I added them to `.env.example` already 👍 I think we might have other places where we don't follow this practice, but we'll need to extract them going forward, to make it more open-source :)

So here I'd rephrase it to something like "make sure to fill in every variable from .env.example to make sure the authentication works"?